docs/design-docs/benchmarks.md

```diff
@@ -244,6 +249,12 @@ result = {
             'throughput-inference-float32': [avg_throughput1, ..., avg_throughputN],
             'throughput-inference-float16': [avg_throughput1, ..., avg_throughputN],
     },
+    'reduce': {
```

Comment:
I don't find the code for the name. Can we change to `reduce_operator`? `reduce` is a little bit strange.

Reply:
It is one member of BenchmarkResult, and has been merged in last PR:
https://github.com/microsoft/superbenchmark/blob/main/superbench/benchmarks/result.py#L34

I will submit separated PR to modify it uniformly.